hw/top_earlgrey/data/chip_conn_testplan.hjson

```diff
@@ -0,0 +1,776 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "chip_conn"
+
+  testpoints: [
+
+    /////////////////////////
+    // AON_TIMER resets    //
+    /////////////////////////
+    {
+      name: AON_TIMER_RST
```

Comment:
nit: Why all caps?

Reply:
Oh, I directly copied it from the connectivity names. I will change them. Thanks Sri.